src/rpc/mempool.cpp

```diff
@@ -633,7 +633,7 @@ static RPCHelpMan gettxspendingprevout()
                                     {"vout", UniValueType(UniValue::VNUM)},
                                 }, /*fAllowNull=*/false, /*fStrict=*/true);
 
-                const uint256 txid(ParseHashO(o, "txid"));
+                const Txid txid = Txid::FromUint256(ParseHashO(o, "txid"));
```

Comment:
(Followup) should check length

Reply:
Looks like `ParseHashO` has a length check internally?

Reply:
Ah, it calls `ParseHashV` which indeed checks that it's 64 characters. So then only the `rest.cpp` line above doesn't check the length (and it's terribly documented).